account_payment_term_multi_day/migrations/8.0.1.0.0/post-migration.py

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 Tecnativa - Vicent Cubells
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openerp.openupgrade import openupgrade
+
+
+def update_paydays(cr):
+    """
+    Propagate paydays field to all lines of payment term.
+    """
+    openupgrade.logged_query(
+        cr,
+        """
+        UPDATE account_payment_term_line aptl
+        SET paydays = (
+            SELECT paydays FROM account_payment_term apt
+            WHERE apt.id = aptl.payment_id)
+    """)
+
+
+@openupgrade.migrate(no_version=True)
+def migrate(cr, version):
+    if not openupgrade.column_exists(
+            cr, 'account_payment_term', 'paydays'):
+        return
+    # If column paydays exists in account_payment_term then module paydays was
```

Comment:
Make everything in the same method, putting this explanation in the docstring, and letting this main method as only a call to each method (one in this case).